### PR TITLE
Compact json download

### PR DESCRIPTION
**Pull request by ashu8912**

### CODE REVIEW

**Comment by ashu8912** [comment]
fixes #772

File: src/components/header/export-modal/renderer.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ class ExportModal extends React.PureComponent<Props, State> {
       content.config = mergeConfig({}, this.props.config, content.config);
     }
 
-    const blob = new Blob([JSON.stringify(content, null, 2)], {
+    const blob = new Blob([JSON.stringify(content, null, 0)], {
```
Comment (domoritz) [inline src/components/header/export-modal/renderer.tsx:105]: Please use json-stringify-pretty-compact instead.

**Review by domoritz — CHANGES_REQUESTED** [review]

File: src/components/header/export-modal/renderer.tsx (hunk 1 above)
Comment (ashu8912) [reply src/components/header/export-modal/renderer.tsx:105]: aah nice we want best of both the worlds, i will add the fix

Comment (ashu8912) [reply src/components/header/export-modal/renderer.tsx:105]: also @domoritz do we want that 0 indentation or the previous default 2 indentation

Comment (domoritz) [reply src/components/header/export-modal/renderer.tsx:105]: We don't need any arguments anymore. Look at other uses of json-stringify-pretty-compact.

**Comment by ashu8912** [comment]
@domoritz fixed and tested as well

**Pull request merged**